dominode-bootstrapper/dominode_bootstrapper/geonodeadmin.py

```diff
@@ -241,15 +292,21 @@ def bootstrap(
             'A group for internal DomiNode users'
         )
         geonode_manager.logout()
+
+    with httpx.Client() as client:
+        geoserver_manager = GeoServerManager(client, base_url, geoserver_username, geoserver_password)
+        for department in DepartmentName:
+            _bootstrap_department_in_geoserver(geoserver_manager, department)
+
```

Comment:
Instead of spawning another `httpx.Client`, modify `GeoServerManager.__init__()` in order to not modify the client's headers. Keep the custom headers in a class attribute of `GeoServerManager` instead and apply them when making each request